src/app/courses-item/courses-item.component.html

```diff
@@ -1,7 +1,24 @@
-<div style="border: 1px solid #000">
-	Id: <span [textContent]="course.id"></span><br>
-	Title: <span [textContent]="course.title"></span><br>
-	CreationDate: <span [textContent]="course.creationDate"></span><br>
-	Duration: <span [textContent]="course.duration"></span><br>
-	Description: <span [textContent]="course.description"></span><br>
+<div class="row no-gutters">
+	<div class="col-auto flex-grow-1">
+		<div class="card">
+			<div class="card-body">
+				<div class="row">
+					<div class="col-6">
+						<div class="card-title h3" [textContent]="course.title"></div>
+					</div>
+					<div class="col-6">
+						<i class="fas fa-calendar-alt"></i> <span [textContent]="course.creationDate"></span><br>
+						<i class="far fa-clock"></i> <span [textContent]="course.duration"></span><br>
+					</div>
+				</div>
+				<div class="description" [textContent]="course.description"></div>
+				<div class="other">
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="col-3">
+		<app-courses-delete [id]="course.id"></app-courses-delete>
```

Comment:
The handler has to be located in the owner of these buttons. I.e. your buttons should emit click events with id.

Reply:
Rewrite with EventEmitter, please